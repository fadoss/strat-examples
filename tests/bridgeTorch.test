games/bridgeTorch

srewrite initial using wrong .
srewrite initial using wiser .
srewrite initial using fastest .
srewrite initial using slowest .

