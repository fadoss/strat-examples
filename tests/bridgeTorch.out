==========================================
srewrite in BRIDGE-SCHECK : initial using wrong .

No solution.
rewrites: 1108
==========================================
srewrite in BRIDGE-SCHECK : initial using wiser .

Solution 1
rewrites: 1184
result Bridge: none | dest pers(1) pers(2) pers(5) pers(8) torch(0)

No more solutions.
rewrites: 1184
==========================================
srewrite in BRIDGE-SCHECK : initial using fastest .

Solution 1
rewrites: 1180
result Bridge: none | dest pers(1) pers(2) pers(5) pers(8) torch(0)

No more solutions.
rewrites: 1875
==========================================
srewrite in BRIDGE-SCHECK : initial using slowest .

No solution.
rewrites: 109
Bye.
